SplunkAWSCloudWatchStreamingMetricsProcessor/lambda_function.py

```diff
@@ -50,16 +50,52 @@ def transform_json_metric_event(metrics):
 	for metric_string in metrics:
 		metric = ast.literal_eval(metric_string)
 		metric_event = copy.deepcopy(metric)
+		
+		
 		# reformat metric_dimensions
 		extracted_dims = {
-			'dims': ','.join([
-				'{}=[{}]'.format(key, value)
+			'dim': ','.join([
+				'{}:{}'.format(key, value)
 				for key, value in metric_event["dimensions"].items()
 			])
 		}
-		metric_event["metric_dimensions"] = extracted_dims["dims"]
+		
+		# reformat values
+		extracted_values = {
+			'val': ','.join([
+				'{}:{}'.format(key, value)
+				for key, value in metric_event["value"].items()
+			])
+		}
+
+		
+		#pull dimensions and values up to top level of metric_event
+		for key, value in metric_event["dimensions"].items():
+			ky = key
+			metric_event[ky] = value
+		
+		for key, value in metric_event["value"].items():
```

Comment:
This leaves `sum` `count` `min` `max` in the `metric_event` and is not being removed later.